packages/parser/lib/css_parser.mly

```diff
@@ -300,17 +404,21 @@ nth_payload:
   /* n-1 */
   /* n */
   /* -n */
-  | n = IDENT WS? combinator = COMBINATOR b = NUMBER {
+  | n = IDENT WS? combinator = combinator b = NUMBER {
     let first_char = String.get n 0 in
     let a = if first_char = '-' then -1 else 1 in
     Nth (ANB ((a, combinator, int_of_string b)))
   }
   /* TODO: Support "An+B of Selector" */
 
+func:
+  | f = FUNCTION { f }
+  | n = not { n }
```

Comment:
not isn't a function, why is added like this?

Reply:
right, but we're not parsing it as a sole function in rule, rather as `:not()`; `| COLON f = func xs = paren_block( ... )`